changelog/64933.fixed.md

```diff
@@ -0,0 +1 @@
+Display a proper error when pki commands fail
```

Comment:
Should we call out that this is a win_pki module thing since we have multiple pki happenings in Salt?

Reply:
Updated the changelog to specify the win_pki module